jvb/src/main/java/org/jitsi/videobridge/ConferenceSpeechActivity.java

```diff
@@ -228,6 +233,32 @@ public AbstractEndpoint getDominantEndpoint()
         }
     }
 
+    /**
+     * Get a list of recent speakers, other than the current dominant one.
+     */
+    public List<String> getRecentSpeakers()
+    {
+        return getSpeakerHistory(1, NUM_RECENT_SPEAKERS);
+    }
+
+    /**
+     * Query whether an endpoint is a recent speaker.
+     */
+    public boolean isRecentSpeaker(AbstractEndpoint endpoint)
```

Comment:
I think we could also do this here:

```
endpointsBySpeechActivity.stream()
    .limit(NUM_RECENT_SPEAKERS)
    .filter(ep -> ep == endpoint)
    .findFirst().orElse(false)
```

Reply:
Oh interesting, I thought stream was specifically supposed to work that way.  I know Kotlin's `asSequence` does and I thought it was the same thing, maybe not.

Reply:
Maybe it was just the overhead of creating the relevant objects, I didn't investigate it that closely.